docs/components/static-map.md

```diff
@@ -0,0 +1,102 @@
+# StaticMap
+
+The simplest `React` wrapper around `MapboxGL`. This is designed to be completely
+static and stateless and will just render a map based on passed in `props`.
```

Comment:
* Mention a use case, i.e. just display a map with no interaction.
* Link to `InteractiveMap`.